prow/tide/tide.go

```diff
@@ -1418,7 +1417,7 @@ func (c *syncController) takeAction(sp subpool, batchPending, successes, pending
 // changedFilesAgent queries and caches the names of files changed by PRs.
 // Cache entries expire if they are not used during a sync loop.
 type changedFilesAgent struct {
-	ghc         githubClient
+	provider    provider
```

Comment:
Q: Use this new keyword in the CLI flags? (e.g., `--changed-files-providers=[github|gerrit|both]` ? Or let it accept a CSV of providers?)

Reply:
if it needs to `providers` makes more sense. As mentioned above that I don't think it's necessary to introduce this